.github/workflows/backport.yml

```diff
@@ -1,16 +1,28 @@
 name: Backport
 on:
-  pull_request:
+  pull_request_target:
     types:
       - closed
       - labeled
 
 jobs:
   backport:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
     name: Backport
     steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
```

Comment:
what is the installation_id ?

Reply:
I don't know what that is. I did see that all the OpenSearch projects use the same value. @VachaShah , what is this?

Reply:
Its the installation id of the opensearch-trigger-bot app.